bundles/org.openhab.binding.nikobus/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -35,14 +36,63 @@
 		</channels>
 
 		<config-description>
-			<parameter name="address" type="text">
+			<parameter-group name="parameterGroupPC">
+				<label>Parameters retrieved from the Niko PC application</label>
```

Comment:
Labels should be short and should follow the convention related to use of uppercase first characters as described in: https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Also check the other labels.

Reply:
Q: for parameter group labels the longer <description> doesn't seem visible in Paper UI. A shortened label may not be that clear to users + there is enough of room for the label to be shown. How short should it be?

Reply:
I don't know how short it should be, but it should roughly be around 3 or 4 words. PaperUI is only 1 UI and other UIs might have less space. PaperUI is also being replaced so other or future UI's might show the description (which I think is should be the case and thus is a missing feature in PaperUI) I would not make the decision on how it currently looks in PaperUI.

Reply:
ok, make sense, thx